.github/workflows/ci.yaml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  basic:
+    name: Run tests on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 1
```

Comment:
added this because test can fail since they are updating app state at the same time